uk.co.compucorp.civicrm.hrcore/gulp/tasks/backstopjs.js

```diff
@@ -199,9 +199,10 @@ function scenariosList () {
 
       scenarios.forEach(function (scenario, index) {
         scenario.credential = scenario.credential || DEFAULT_CREDENTIAL;
+        scenario.onBeforeScript = 'initialise';
```

Comment:
"init" (so you also avoid the controversy between initialize and initialise)

Reply:
Haha, nice point